build-scripts/ubuntu-1604/build-3rd-parties.sh

```diff
@@ -83,3 +83,9 @@ build_from_pypi intervaltree 2.1.0
 build_from_pypi jsonpickle 0.9.6
 # TODO: add libsnappy dependency for python-rocksdb package
 build_from_pypi python-rocksdb 0.6.9
+
+build_from_pypi wcwidth 
+build_from_pypi ujson 1.33
+build_from_pypi leveldb 
+build_from_pypi msgpack-python 0.4.6
+build_from_pypi indy-crypto
```

Comment:
indy-crypto should not be there, as this is already defined in setup.py, and there is corresponding deb available.